workflow/rules/process/assign-gdp.smk

```diff
@@ -0,0 +1,91 @@
+"""Assign elements for each box
+
+"""
+
+out_assigngdp = (
```

Comment:
Here, and in various other places, it's helpful to separate all words in variable/function names with _s so this would be `out_assign_gdp`, the rule would be `rule process_assign_gdp`, etc.